soc/arm/ti_k3/j721e/soc.h

```diff
@@ -0,0 +1,10 @@
+/* Copyright (C) 2023 BeagleBoard.org Foundation
+ * Copyright (C) 2023 S Prashanth
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _TI_K3_J721E_R5_SOC_H_
+#define _TI_K3_J721E_R5_SOC_H_
+
+#endif /* _TI_K3_J721E_R5_SOC_H_ */
```

Comment:
remove if not needed

Reply:
soc.h is included in include/zephyr/arch/arm/aarch32/cortex_a_r/cmsis.h

Reply:
Cortex-M/A/R require soc.h, even if empty because of how CMSIS is integrated. Ref. https://github.com/zephyrproject-rtos/zephyr/issues/57246